NEWS.md

```diff
@@ -1,3 +1,7 @@
+# blastula 0.3.3
```

Comment:
I believe it should be 0.3.4 now or any other header for development version now that 0.3.3 has been release